RELEASE_NOTES.md

```diff
@@ -9,6 +9,12 @@ This page describes the noteworthy improvements provided by each release of Ecli
 * 📅 Release Date (tentative): End of July 2020
 * All changes: https://github.com/eclipse/wildwebdeveloper/compare/0.10.0...0.11.0
 
+#### Embedded NodeJS
+
+Embedded Node.js from Wild Web Developer is now available. Unless the `"org.eclipse.wildwebdeveloper.nodeJSLocation"` system property is set and user's system suits the supported OSs (Linux, MacOS and Win32) and architectures (x86_64), the embedded version of NodeJS will be used by Language Servers and Node Debugger. 
```

Comment:
I would word it something like: Node.js is now embedded into Wild Web Developer.

Reply:
Also, the second time Node.js is mentioned, it's written as `NodeJS`. I would keep it consitent by writting `Node.js`